h/services/flag_count.py

```diff
@@ -19,7 +19,7 @@ def flag_count(self, annotation):
         """
         return (
             self._session.query(sa.func.count(Flag.id))
-            .filter_by(annotation_id=annotation.id)
+            .filter(Flag.annotation_id == annotation.id)
```

Comment:
I not 100%  sure but I think it could be a result of this:

https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#a-select-statement-is-no-longer-implicitly-considered-to-be-a-from-clause

The exception trace seemed to indicate it was trying to find "anotation_id" on a count() type of object so hinting what the actual table being filtered with `Flag` seemed to help